src/components/Application-Body/header/Header.tsx

```diff
@@ -63,25 +68,28 @@ const Header: React.FC<HeaderOwnProps> = (props): JSX.Element => {
                 onChange={onSearchValueChange}
                 value={searchValue}
             />
-                
-            <Button
-                className={classes.addNewPostButton}
-                variant="contained"
-                onClick={() => setOpenAddDialog(true)}
-                ref={addButtonRef}
-            >
-                <Typography
-                    variant='h6'
-                >
-                    + פרסום תפקיד חדש
-                </Typography>
-            </Button>
-            { openAddDialog && allSelectOptions && <PostNewJob 
-                allSelectOptions={allSelectOptions}
-                fetchAllAdsAfterPost={fetchAllAdsAfterPost}
-                closeDialog={onCloseDialog} />}
+            { userStore.isRamad &&
```

Comment:
The comment above... I would decide what to render upon the `userStore.isRamad`... Assuming the header file is in `PersonalZone file`, like we talked about.

Reply:
I will do in my MR